releaser.py

```diff
@@ -39,8 +39,7 @@ def template_substitute(contents):
     today = datetime.date.today()
     fmt_today = today.strftime("%B %d, %Y")
 
-    header_line = f"{name} {version} ({fmt_today})"
-    res += f"{header_line}\n{'=' * len(header_line)}\n\n"
+    res += f"# {name} {version} ({fmt_today})\n\n"
```

Comment:
`releaser.py` and `changelog.py` are changed manually by me, to make sure new changelogs are written correctly